packages/lesswrong/components/localGroups/LocalGroupPage.jsx

```diff
@@ -6,114 +6,95 @@ import { Posts } from '../../lib/collections/posts';
-    marginTop: "93px",
+  groupInfo: {
+    ...sectionFooterLeftStyles
   },
-
   groupName: {
     ...theme.typography.headerStyle,
     fontSize: "30px",
-
     marginTop: "0px",
     marginBottom: "0.5rem"
   },
   groupSubtitle: {
+    marginBottom: theme.spacing.unit * 2
+  },
+  leftAction: {
+    [theme.breakpoints.down('xs')]: {
+      textAlign: 'left'
+    }
   },
   groupLocation: {
     ...theme.typography.body2,
-
     display: "inline-block",
     color: "rgba(0,0,0,0.7)",
   },
   groupLinks: {
     display: "inline-block",
-    marginTop: "-7px",
-    marginLeft: "10px",
-    marginBottom: "20px",
   },
   groupDescription: {
-    marginLeft: "24px",
     marginBottom: "30px",
-
     [theme.breakpoints.down('xs')]: {
       marginLeft: 0
     }
   },
-
   groupDescriptionBody: {
     ...postBodyStyles(theme),
+    padding: theme.spacing.unit,
   }
 });
 
 class LocalGroupPage extends Component {
-  renderTitleComponent = () => {
-    const { classes } = this.props;
+  render() {
+    const { classes, document:group, currentUser } = this.props;
     const { groupId } = this.props.params;
-    const group = this.props.document;
+    const { CommunityMapWrapper, SingleColumnSection, SectionTitle, GroupLinks, PostsList2, Loading, 
+      SectionButton, SubscribeTo, SectionFooter, GroupFormLink } = Components
+    if (!group) return <Loading />
+    const { html = ""} = group.contents || {}
+    const htmlBody = {__html: html}
+    
+    const { googleLocation: { geometry: { location } }} = group;
     return (
-      <div className={classes.groupSidebar}>
-        {this.props.currentUser
-          && <Components.SectionSubtitle>
-            <Components.SubscribeTo document={group} />
-          </Components.SectionSubtitle>}
-        {Posts.options.mutations.new.check(this.props.currentUser)
-          && <Components.SectionSubtitle>
-            <Link to={{pathname:"/newPost", query: {eventForm: true, groupId}}}>
-              Create new event
-            </Link>
-          </Components.SectionSubtitle>}
-        {Posts.options.mutations.new.check(this.props.currentUser)
-          && <Components.SectionSubtitle>
-            <Link to={{pathname:"/newPost", query: {groupId}}}>
-              Create new group post
-            </Link>
-          </Components.SectionSubtitle>}
-        {Localgroups.options.mutations.edit.check(this.props.currentUser, group)
-          && <Components.GroupFormLink documentId={groupId} label="Edit group" />}
+      <div className={classes.root}>
+        <CommunityMapWrapper
+          terms={{view: "events", groupId: groupId}}
+          groupQueryTerms={{view: "single", groupId: groupId}}
+          mapOptions={{zoom:11, center: location, initialOpenWindows:[groupId]}}
+        />
+        <SingleColumnSection>
+          <SectionTitle title={group.name}>{currentUser && <SectionButton><SubscribeTo document={group} /></SectionButton>}</SectionTitle>
```

Comment:
I'd personally prefer this broken up into multiple lines, ala
<SectionTitle title={group.name}>
{ blah}
</SectionTitle>